### ISSUE TITLE
Constant crashes

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#30506]
(WEB REPORT BY: soyconsumer REMOTE: 206.221.180.138:7777)
# Revision
3cd2eb209be77ae7b7d8c434cf98a629f6a3e45d

Game crashes multiple times a round, doesn't just happen to me as people bitch in OOC
I'm using BYOND version 513.1542

**Comment by bathosbathtime** [comment]
Happens to me as well, on Revision: 3cd2eb209be77ae7b7d8c434cf98a629f6a3e45d
byond version 514.1557

**Comment by Eneocho** [comment]
I swear lummox must've added some cryptominers or some bullshit to latest 514, because it runs like ass and crashes constantly.

**Comment by SonixApache** [comment]
probably doesn't matter but might be good to include map and round length

**Comment by Eneocho** [comment]
Happened to me as well.
Byond 513 latest.

**Comment by Exxion** [comment]
Have you considered updating to 514

**Comment by Eneocho** [comment]
> 
> 
> Have you considered updating to 514

I had actually downgraded from latest 514 since it ran like ass and crashed constantly. It ran fine for a while on 513, but one specific round yesterday caused me to crash randomly.

**Comment by Exxion** [comment]
Yeah, it does run pretty slowly for whatever reason. But I've heard that since the server was updated to latest, earlier client versions have tended to crash. Did you downgrade before or after the server was updated?

**Comment by Eneocho** [comment]
Downgraded before the serb update.

**Comment by Exxion** [comment]
Well then maybe try 514 again now that the server's updated
